package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "npm-docker-elasticsearch",
+  "version": "1.0.0",
+  "description": "Elasticsearch with GCS Fuse",
+  "main": "Dockerfile",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/npm/npm-docker-elasticsearch.git"
```

Comment:
prefer plain git repo urls?

Reply:
this is what npm gave me when i ran `npm init`

Reply:
fixed it